app/controllers/comments_controller.rb

```diff
@@ -1,5 +1,6 @@
 class CommentsController < ApplicationController
   before_action :set_comment, only: %i[ show edit update destroy ]
+  before_action :is_an_authorized_user, only: [:destroy, :create]
```

Comment:
Nice job on adding an authorization check to make sure that only authorized users can create and destroy comments.